assets/l10n/app_en.arb

```diff
@@ -479,5 +479,14 @@
       "senderFullName": {"type": "String", "example": "Alice"},
       "numOthers": {"type": "int", "example": "4"}
     }
+  },
+  "typingIndicator": "{numPeople, plural, =1{{typist} is typing} =2{{typist} and {otherTypist} are typing} other{Several people are typing}}",
```

Comment:
Thanks! Those are some interesting distinctions. `=2` seems to be used for English after modifying the test case.

We can split this into 3 separate strings (or just two strings equivalent to the "=1" and "other" here, respectively) without the "plural" feature.

Reply:
> `=2` seems to be used for English after modifying the test case.

Huh, surprising.

In that case I'd guess that in Chinese it'd use the `=1` and `=2` forms too, if present… but a translator UI, like that in Transifex, generally wouldn't ask the Chinese translators for such forms, because in the normal grammatical behavior of the language (more precisely, in its behavior as recorded in the CLDR database, shown in that giant table at my third link) there's no distinction that would call for such forms. So we'd end up with only an "other" form in our translations.

And I'd expect that in Russian it'd still behave the way I predicted above, using the `=1` form more broadly than this bit of program logic intends.

Reply:
Cross-posted a version of the above to #mobile-team, and then added further findings:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/i18n.20plurals.20.2F.20grammatical.20number/near/1848780